roles/node_prep/tasks/40_bridge.yml

```
@@ -80,7 +80,7 @@
         state: present
       when: (not enable_virtualmedia) and
             ((ipv4_provisioning|bool) or (not ipv6_enabled|bool) or
-            ((release_version.split('.')[0]|int == 4) and (release_version.split('.')[1]|int == 3)))
+            (release_version is ansible.builtin.version('4.3', '=='))
```

same here, I believe we are missing to close a parenthesis at the end of this line, to group all the `or` conditions.
```suggestion
            (release_version is ansible.builtin.version('4.3', '==')))
```

---

True, changed